specification/purview/data-plane/Azure.Analytics.Purview.PDS/preview/2021-01-01-preview/pds.json

```diff
@@ -0,0 +1,899 @@
+          }
+        },
+        "x-ms-examples": {
+          "PolicyEvents3_Get": {
+            "$ref": "./examples/PolicyEvents3_Get.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AccountModelBase": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "location": {
+          "type": "string"
+        },
+        "tenantId": {
+          "type": "string"
+        },
+        "gatewayEndpoint": {
+          "type": "string"
+        }
+      }
+    },
+    "Consent": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "boolean"
+        },
+        "properties": {
+          "$ref": "#/definitions/Properties"
+        },
+        "excludedScopes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "DatasourcePreferencesResponse": {
+      "type": "object",
+      "properties": {
+        "consent": {
+          "$ref": "#/definitions/Consent"
+        },
+        "accountInfo": {
+          "$ref": "#/definitions/AccountModelBase"
+        }
+      }
+    },
+    "ErrorModel": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorModel"
+          }
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorModel"
+        }
+      }
+    },
+    "PolicyElement": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "scopes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "kind": {
+          "type": "string"
+        },
+        "updatedAt": {
+          "type": "string"
+        },
+        "version": {
+          "format": "int64",
+          "type": "integer"
+        },
+        "elementJson": {
+          "type": "string"
+        }
+      }
+    },
+    "PolicyElementEventModel": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "scopes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "kind": {
+          "type": "string"
+        },
+        "updatedAt": {
+          "type": "string"
+        },
+        "version": {
+          "format": "int64",
+          "type": "integer"
+        },
+        "elementJson": {
+          "type": "string"
+        },
+        "eventType": {
+          "type": "string"
+        }
```

Comment:
This is another property that looks like it probably should be an enum.

Reply:
Hi Mike, Pasting the same comment here.
We will do this change in the GA API.
But I am afraid that we cannot accommodate this change in this Preview version.
Resolving this comment for now.